app/lib/sharezone_plus/page/sharezone_plus_page_controller.dart

```diff
@@ -79,8 +79,19 @@ class SharezonePlusPageController extends ChangeNotifier {
   }
 
   Future<void> _buyOnWeb() async {
+    // The URL is used to redirect the user back to the web app after the
+    // payment is completed or canceled. We only use the base URL because we a
+    // whitelist for the allowed URLs.
```

Comment:
Last sentence is incomplete. Also I don't follow the argument, I mean the backend could also just match the base url if a longer url is send, no? I guess the argument is so that we have to do less parsing in the backend we only send the base url, correct?

Reply:
Changed the backend that it only needs to match the beginning of the URL ✅